net-analyzer/netdata/netdata-2.5.3.ebuild

```diff
@@ -0,0 +1,196 @@
+REQUIRED_USE="
+	mysql? ( python )
+	python? ( ${PYTHON_REQUIRED_USE} )"
+
+# most unconditional dependencies are for plugins.d/charts.d.plugin:
+RDEPEND="
+	acct-group/netdata
+	acct-user/netdata
+	app-alternatives/awk
+	app-arch/brotli:=
+	app-arch/lz4:=
+	app-arch/zstd:=
+	app-misc/jq
+	>=app-shells/bash-4:0
+	dev-libs/json-c:=
+	dev-libs/libpcre2:=
+	dev-libs/libuv:=
+	dev-libs/libyaml
+	dev-libs/protobuf:=
+	|| (
+		net-analyzer/openbsd-netcat
+		net-analyzer/netcat
+	)
+	net-libs/libwebsockets
+	net-misc/curl
+	net-misc/wget
+	sys-apps/util-linux
+	sys-libs/libcap
+	sys-libs/zlib:=
+	cups? ( net-print/cups )
+	dbengine? (
+		dev-libs/judy
+		dev-libs/openssl:=
+	)
+	ipmi? ( sys-libs/freeipmi )
+	jemalloc? ( dev-libs/jemalloc:= )
+	mongodb? ( dev-libs/mongo-c-driver )
+	nfacct? (
+		net-firewall/nfacct
+		net-libs/libmnl:=
+	)
+	nodejs? ( net-libs/nodejs )
+	postgres? ( net-analyzer/netdata-go-plugin )
+	prometheus? (
+		app-arch/snappy:=
+	)
+	python? (
+		${PYTHON_DEPS}
+		$(python_gen_cond_dep 'dev-python/pyyaml[${PYTHON_USEDEP}]')
+		mysql? ( $(python_gen_cond_dep 'dev-python/mysqlclient[${PYTHON_USEDEP}]') )
+	)
+	xen? (
+		app-emulation/xen-tools
+		dev-libs/yajl
+	)
+	systemd? ( sys-apps/systemd )"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	virtual/pkgconfig
+	mold? ( sys-devel/mold )
+"
+
+FILECAPS=(
+	'cap_dac_read_search,cap_sys_ptrace+ep'
+	'usr/libexec/netdata/plugins.d/apps.plugin'
+	'usr/libexec/netdata/plugins.d/debugfs.plugin'
+)
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+	linux-info_pkg_setup
+}
+
+#PATCHES=(
+#	"${FILESDIR}"/${PN}-dlib-global_optimization-add-template-argument-list.patch
+#)
+
+src_configure() {
+	local mycmakeargs=(
+		-DNETDATA_DLIB_SOURCE_PATH="${WORKDIR}/dlib-${DLIB_VER}"
+		-DCMAKE_DISABLE_FIND_PACKAGE_Git=ON
+		-DCMAKE_INSTALL_PREFIX=/
+		-DFETCHCONTENT_FULLY_DISCONNECTED=ON
+		-DFETCHCONTENT_UPDATES_DISCONNECTED=ON
+		-DENABLE_BUNDLED_JSONC=OFF
+		-DENABLE_BUNDLED_PROTOBUF=OFF
+		-DENABLE_BUNDLED_YAML=OFF
+		-DENABLE_DASHBOARD=OFF # handle manually in install phase
+		-DENABLE_DBENGINE=$(usex dbengine)
+		-DENABLE_EXPORTER_MONGODB=$(usex mongodb)
+		-DENABLE_EXPORTER_PROMETHEUS_REMOTE_WRITE=$(usex prometheus)
+		-DENABLE_JEMALLOC=$(usex jemalloc)
+		-DENABLE_LIBBACKTRACE=OFF
+		-DENABLE_MIMALLOC=OFF
+		-DENABLE_ML=ON
+		-DENABLE_PLUGIN_APPS=ON
+		-DENABLE_PLUGIN_CGROUP_NETWORK=ON
+		-DENABLE_PLUGIN_CHARTS=ON
+		-DENABLE_PLUGIN_CUPS=$(usex cups)
+		-DENABLE_PLUGIN_DEBUGFS=ON
+		-DENABLE_PLUGIN_EBPF=OFF # bundles libbpf
+		-DENABLE_PLUGIN_GO=OFF
+		-DENABLE_PLUGIN_FREEIPMI=$(usex ipmi)
+		-DENABLE_PLUGIN_NFACCT=$(usex nfacct)
+		-DENABLE_PLUGIN_OTEL=OFF
+		-DENABLE_PLUGIN_PERF=ON
+		-DENABLE_PLUGIN_PYTHON=$(usex python)
+		-DENABLE_PLUGIN_SLABINFO=ON
+		-DENABLE_PLUGIN_SYSTEMD_JOURNAL=$(usex systemd)
+		-DENABLE_PLUGIN_SYSTEMD_UNITS=$(usex systemd)
+		-DENABLE_PLUGIN_XENSTAT=$(usex xen)
+		-DUSE_LTO=$(usex lto)
+		-DUSE_MOLD=$(usex mold)
```

Comment:
Do these make sense as use flags? Like do they do more than set CFLAGS, LDFLAGS and LD? You could just use $(tc-is-lto) and $(c-ld-is-mold) to respect the users toolchain variables instead.